### PR TITLE
Add a command to rescan the workspace

### PR DESCRIPTION
**Pull request by bobbrow**
Addresses https://github.com/microsoft/vscode-cpptools-api/issues/11.
Should we call it "rescan folder" or "rescan workspace"?

### CODE REVIEW

**Comment by sean-mcmanus** [comment]
rescan folder seems fine, since the command is just for a single workspace folder and not the entire workspace.

File: Extension/src/LanguageServer/extension.ts [hunk 1]
```diff
@@ -912,6 +913,11 @@ function onLogDiagnostics(): void {
     clients.ActiveClient.logDiagnostics();
 }
 
+function onRescanFolder(): void {
+    onActivationEvent();
+    clients.ActiveClient.rescanFolder();
```
Comment (bobbrow) [inline Extension/src/LanguageServer/extension.ts]: need to think about multi-root

**Review by Colengms — APPROVED** [review]

File: Extension/src/LanguageServer/extension.ts (hunk 1 above)
Comment (bobbrow) [reply Extension/src/LanguageServer/extension.ts]: I talked with @sherryyshi about this and we decided to go with "rescan workspace"

File: Extension/package.nls.zh-cn.json [hunk 2]
```diff
@@ -18,5 +18,6 @@
     "c_cpp.command.showParsingCommands.title": "查看解析命令",
     "c_cpp.command.takeSurvey.title": "调查问卷",
     "c_cpp.command.buildAndDebugActiveFile.title": "生成和调试当前文件",
-    "c_cpp.command.logDiagnostics.title": "记录诊断"
+    "c_cpp.command.logDiagnostics.title": "记录诊断",
+    "c_cpp.command.rescanWorkspace.title": "重新分析工作区"
```
Comment (xisui-MSFT) [inline Extension/package.nls.zh-cn.json]: I think "重新扫描工作区" would be a better translation.

Comment (bobbrow) [reply Extension/package.nls.zh-cn.json]: thanks! :)

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]
The commands should all "title cased" to match the other VS Code command styles.

Some of our existing commands had the incorrect "sentence" casing and no one noticed.

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**